rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -270,8 +268,7 @@ object Jupyter {
 
   def show(xLabel: String, yLabel: String, title: String, plots: Plot*)(
```

Comment:
what if instead of getting rid of OutputHandler from show, we kept this as is but then also provided `draw` versions that return a `Drawable`?